Fix Kostal Piko Legacy in standby

premultiply opened the pull request:
Fix https://github.com/evcc-io/evcc/discussions/3496#discussioncomment-2935810

premultiply committed:
fix template

andig commented on templates/definition/meter/kostal-piko-legacy.yaml:
```diff
@@ -23,5 +23,5 @@ render: |
       user: {{ .user }}
       password: {{ .password }}
-    regex: '(?s)aktuell.*?(\d+)<'
+    regex: '(?s)aktuell<\/td>\s*<td[^>]*>\s*(\d+)'
```
Das wäre dann auch (mit 2x non-greedy Modifier):

    regex: '(?s)aktuell.*?>.*?>\s*(\d+)'

andig replied:
Ahh, hast Recht. Non-greedy, aber es muss insgesamt trotzdem matchen… Jetzt würde es einen Test dieser Regex brauchen? Probehalber mergen oder hast Du ein Testexemplar?

andig commented:
`default` ist im nightly vorhanden- klappts damit?

premultiply committed:
add default value

Return 0 if inverter is in standby and the regex does not match on string value "x x x".

premultiply commented:
Kann ich so direkt selbst nicht testen. Test via Nightly beim Anwender.
